spec/gameobj-data/skin_spec.rb

```diff
@@ -324,6 +321,60 @@
       end
     end
 
+    describe "bundle skin oddities - recognize single skin, but not bundle" do
+      [
+        %{elongated triton spine},
+        %{iridescent triton hide},
+        %{soft red firebird feather},
+        %{curved gold-flecked claw},
+      ].each do |skin_name|
+        it "doesn't recognizes single #{skin_name} as a skin" do
+          skin = GameObjFactory.item_from_name(skin_name)
+          expect(skin.type).to eq "skin"
+          expect(skin.sellable.to_s).to eq "furrier"
+        end
+
+        pluralized_skin_name = if skin_name.end_with? "s"
+                                skin_name
+                              else
+                                "#{skin_name}s"
+                              end
+
+        it "recognizes bundle of #{pluralized_skin_name} as a skin" do
```

Comment:
shouldn't this be doesn't?

Reply:
Yep, forgot to update this when copied again to do reverse lookup.